src/sections/Community/Handbook/faq.js

```diff
@@ -17,43 +17,43 @@ const Faq = () => {
         <Container>
           <div className="content">
             <h2>General FAQs:</h2><br />
-            <h4>Q: Are Layer5’s solutions open source?</h4>
-            <p>A: Yes, all Layer5 projects are licensed under Apache V2. Layer5 also offers extensions to the Meshery project in which Enterprise-centric functionality is commercially offered and supported.</p>
+            <h3>Q: Are Layer5’s solutions open source?</h3>
+            <p><strong>A:</strong> Yes, all <strong>Layer5</strong> projects are licensed under Apache V2. <strong>Layer5</strong> also offers extensions to the Meshery project in which Enterprise-centric functionality is commercially offered and supported.</p>
```

Comment:
Also, I don't think making Layer5 bold creates an impact while looking at the FAQ page in the preview

- If you are really looking to change it into bold please do the same with all of the `Layer5` words on the page. Just to be consistent

Reply:
Okay Sure , Thanks @Mohith234 !